### PR TITLE
document container registry setup for Minikube and AKS

### PR DESCRIPTION
**Pull request by bacongobbler**
This documents the new-and-improved setup process to use Draft with and without a container registry.

### CODE REVIEW

**Comment by squillace** [comment]
Looking

File: docs/install-advanced.md [hunk 1]
```diff
@@ -7,6 +7,26 @@ In certain situations, users are given a Kubernetes cluster with tighter securit
 
 This document explains some of these situations as well as how it can be handled using draft.
 
+## Drafting in the Cloud
```
Comment (michelleN) [inline docs/install-advanced.md]: love this heading. so cute!

File: docs/install-advanced.md [hunk 2]
```diff
@@ -7,6 +7,26 @@ In certain situations, users are given a Kubernetes cluster with tighter securit
...
+
+When using Draft with cloud-provided Kubernetes solutions like [Azure Container Service](https://azure.microsoft.com/en-us/services/container-service/), we need a way to distribute the built image across the cluster. A container registry allows all nodes in the Kubernetes cluster to pull the images we build using Draft, and we have a way for our local Docker daemon to distribute the built image.
+
+For this example, we want to push images to our registry sitting at `myregistry.azurecr.io`, and pull those images down to the Kubernetes cluster from that same registry. To do that, we run
+
+```shell
+$ draft config set registry myregistry.com
```
Comment (michelleN) [inline docs/install-advanced.md]: should this be myregistry.azurecr.io?

Comment (bacongobbler) [reply docs/install-advanced.md]: yes, you're right

**Comment by bacongobbler** [comment]
fixed. Can I get an LGTM? This should help new users trying out Draft v0.12.0 for the first time.

**Comment by bacongobbler** [comment]
Thanks Radu!

**Pull request merged**